backend/schemas/source/excel/templates/federal-awards-expended-template.jsonnet

```diff
@@ -107,9 +115,11 @@ local open_ranges_defns = [
     'If Other Cluster, Enter Other Cluster Name',
     'other_cluster_name',
   ],
-  // 20230525 HDMS FIXME: A formula to auto calculate federal_program_total in the excel is missing (see instructions in census template)!!!
   [
     Sheets.open_range {
+      keep_locked: true,
+      format: 'dollar',
+      formula: '=SUMIFS(E:E,V:V,V{0})',
```

Comment:
Also a place for named ranges? Or, do we need to use the cell refs, because of the `{0}`?

Reply:
I believe we could change E:E and V:V to named ranges but not V{0}. Also, I will have to try this in each case as I am not sure how excel react to named range in formulas versus cell refs